unreal_asset/unreal_asset_exports/src/lib.rs

```diff
@@ -0,0 +1,7 @@
+//! Unreal asset exports
+
+use unreal_asset_base::*;
```

Comment:
Why is exports a module? Can't the contents be lifted to the crate root? These imports are useless?

Reply:
mostly better lint performance, these imports are required so imports don't have to be changed in every source file